lib/praxis/plugins/praxis_mapper_plugin.rb

```diff
@@ -140,8 +151,15 @@ module Controller
 
       module Statistics
 
-        def self.log(identity_map, log_stats)
+        def self.log(request, identity_map, log_stats)
           return if identity_map.nil?
+          return if request.silence_mapper_stats == true
+          if identity_map.queries.empty?
+            self.to_logger "No database interactions observed."
+            return
+          end
```

Comment:
Did you mean to leave this in? I thought you wanted consistent output when no interactions occurred?